scale/job/models.py

```diff
@@ -2355,12 +2415,12 @@ class TaskUpdate(models.Model):
     """
 
     job_exe = models.ForeignKey('job.JobExecution', on_delete=models.PROTECT)
-    task_id = models.CharField(max_length=50)
-    status = models.CharField(max_length=50)
+    task_id = models.CharField(max_length=250)
```

Comment:
I think having a lax length is good in case something really goes sideways; my initial restrictions were just way too strict. I think max 250 is appropriate for these fields (if the length is over 250 for any of these, something IS really wrong and we should see it as an error).